app/assets/javascripts/common/reports.js

```diff
@@ -432,8 +432,7 @@ Reports = function (withLtfu) {
     const controlledGraphRate = withLtfu
       ? data.controlWithLtfuRate
       : data.controlRate;
-
-    const config = {
+    let config = {
```

Comment:
I need to be able to overwrite the config in cases where the goal lines are used. The original config and the additional 'goal line' config snippet is merged and the full config is returned.